### DISCUSSION TITLE
Scenariusz AAD B2B MFA multi-tenant

### DISCUSSION THREAD

**Discussion by pawelklimczyk** [AzureCommunityPL/group#309] [Pomoc]
Czy da sie tak skonfigurowac MFA (albo coś innego) na Tenant 1 zeby uzytkownicy z innych tenantow (jak Tenant 2) musieli użyć MFA jeżeli chcą użyć aplikacji "App". Dla Tenant 1 działa to przez utworzenie Conditional Policy

W skócie chodzi o to żeby wymusić MFA jeżeli jakikolwiek user chce się zalogować (niezależnie od konfiguracji na jego tenancie)

![image](https://user-images.githubusercontent.com/1084668/123236648-45de7580-d4dd-11eb-91e0-a48f21660800.png)

**Comment by TomaszOledzki** [comment]
Da się. Miałem kilka takich case już.

Konto z Tenant 2 dodane jako guest account w Tenant 1 i miało wymuszone MFA przy uwierzytelnianiu w AAD z Tenant 1. Do tego w AAD w Tenant 2 nie było MFA.

**Reply by pawelklimczyk** [reply]
Czyli jezeli mam Tenant 2, Tenant 3, Tenant n... i kazde ma po 1000 kont, to wszystkie trzeba dodac jako "guest-account" w Tenant 1?

Najbardziej satysfakcjonowalo by mnie dodanie w Tenant 1 calego Tenant 2 i skonfigurowanie MFA dla Tenant 2. O ile sie tak da...

**Reply by pizang** [reply]
Tak ale też i nie :)

To co chcesz użyć to Azure AD - Identity Governance - Connected organizations. Tutaj dodasz zewnętrzną organizację po jej domenie. Później w AAD tworzysz Access Packages. W ogólnym przypadku ustawia się wtedy osobę aprobującą dostęp ale można też to pominąć. Jeżeli masz więc zaufaną domenę to wszyscy użytkownicy z tej domeny będą mogli zażądać lub pobrać dostęp do danych zasobów.

https://www.youtube.com/watch?v=zaaKvaaYwI4

**Comment by PawelHaracz** [comment]
Możesz zobaczyć access policy czy nie rozwiąże Twojego problemu
https://docs.microsoft.com/en-us/azure/active-directory/conditional-access/concept-conditional-access-cloud-apps
